src/vivarium_gates_mncnh/components/children.py

```diff
@@ -128,4 +139,29 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             index=index,
         )
 
+        # Update gestational age for partial term pregnancies
+        pregnancy_outcomes = self.population_view.subview([COLUMNS.PREGNANCY_OUTCOME]).get(
+            index
+        )
+        partial_term_idx = pregnancy_outcomes.index[
+            pregnancy_outcomes[COLUMNS.PREGNANCY_OUTCOME]
+            == PREGNANCY_OUTCOMES.PARTIAL_TERM_OUTCOME
+        ]
+        new_children.loc[
+            partial_term_idx, COLUMNS.GESTATIONAL_AGE
+        ] = self.get_partial_term_gestational_age(
+            partial_term_idx,
+        )
+
```

Comment:
@rmudambi To be very clear here, this does mean that this column will no longer be in sync with our `birth_weight_exposure` column created by the LBWSG component. I think that could be pretty confusing if we aren't clear about this.

Reply:
Per discussion this morning: Removing extra birth weight and gestational age columns and adding a partial term pregnancy column.